markhor_bringup/launch/tpv_controller.launch

```diff
@@ -0,0 +1,6 @@
+<launch>
+    <node name="microcontroller_command_node"        pkg="rosserial_python"      type="serial_node.py">
```

Comment:
Are the spaces necessary ? And if not, I think we should follow the same standard that we did for the other launch files.